src/project/manifest/activation.rs

```diff
@@ -1,7 +1,10 @@
 use serde::Deserialize;
+use std::collections::HashMap;
 
 #[derive(Default, Clone, Deserialize, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct Activation {
     pub scripts: Option<Vec<String>>,
+    /// Environment variables to set before running the scripts.
+    pub env: Option<HashMap<String, String>>,
```

Comment:
I think in the future it might become important to retain the order of insertion. So perhaps using an `IndexMap` is more appropriate.